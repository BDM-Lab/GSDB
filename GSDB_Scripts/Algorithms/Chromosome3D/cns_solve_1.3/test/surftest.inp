remarks SURFTEST.INP
remarks TEST OF THE RICHARD'S ACCESSIBLE SURFACE AND CONTACT PROPERTIES USING
remarks A SHORTENED PTI TO SPEED THE RUN.


topology @CNS_TOPPAR:protein.top  @CNS_TOPPAR:water.top end
parameter @CNS_TOPPAR:protein.param  @CNS_TOPPAR:water.param end

segment 
   name=m1 
   chain      
      @CNS_TOPPAR:protein.link      
      sequence       
         arg pro asp phe cys leu
      end
   end 
end

segment
  name=m2
  chain
     @CNS_TOPPAR:protein.link
     sequence cys end
  end
end

patch disu reference 1 ( atom m1 5 * ) reference 2 ( atom m2 1 * )  end

segment 
    name=solv    
    molecule 
       name=TIP  number=1 
    end 
end 

coor 
ATOM      1  HT1 ARG     1      26.033  27.513  -3.554  1.00  0.00      M1  
ATOM      2  HT2 ARG     1      26.877  28.308  -2.405  1.00  0.00      M1  
ATOM      3  N   ARG     1      26.521  27.417  -2.687  1.00  0.00      M1  
ATOM      4  HT3 ARG     1      27.281  26.776  -2.801  1.00  0.00      M1  
ATOM      5  CA  ARG     1      25.603  26.909  -1.661  1.00  0.00      M1  
ATOM      6  CB  ARG     1      24.610  25.885  -2.248  1.00  0.00      M1  
ATOM      7  CG  ARG     1      25.171  24.458  -2.379  1.00  0.00      M1  
ATOM      8  CD  ARG     1      24.124  23.541  -2.979  1.00  0.00      M1  
ATOM      9  NE  ARG     1      23.644  24.058  -4.269  1.00  0.00      M1  
ATOM     10  HE  ARG     1      22.790  24.577  -4.256  1.00  0.00      M1  
ATOM     11  CZ  ARG     1      24.299  23.863  -5.432  1.00  0.00      M1  
ATOM     12  NH1 ARG     1      25.434  23.152  -5.473  1.00  0.00      M1  
ATOM     13 HH11 ARG     1      25.807  22.758  -4.633  1.00  0.00      M1  
ATOM     14 HH12 ARG     1      25.908  23.016  -6.342  1.00  0.00      M1  
ATOM     15  NH2 ARG     1      23.797  24.379  -6.539  1.00  0.00      M1  
ATOM     16 HH21 ARG     1      24.265  24.240  -7.412  1.00  0.00      M1  
ATOM     17 HH22 ARG     1      22.950  24.909  -6.504  1.00  0.00      M1  
ATOM     18  C   ARG     1      26.256  26.217  -0.466  1.00  0.00      M1  
ATOM     19  O   ARG     1      27.268  25.519  -0.612  1.00  0.00      M1  
ATOM     20  N   PRO     2      25.669  26.420   0.698  1.00  0.00      M1  
ATOM     21  CD  PRO     2      24.639  27.414   0.964  1.00  0.00      M1  
ATOM     22  CA  PRO     2      26.191  25.807   1.926  1.00  0.00      M1  
ATOM     23  CB  PRO     2      25.364  26.408   3.014  1.00  0.00      M1  
ATOM     24  CG  PRO     2      24.531  27.554   2.480  1.00  0.00      M1  
ATOM     25  C   PRO     2      26.248  24.283   1.838  1.00  0.00      M1  
ATOM     26  O   PRO     2      25.462  23.661   1.112  1.00  0.00      M1  
ATOM     27  N   ASP     3      27.177  23.707   2.577  1.00  0.00      M1  
ATOM     28  H   ASP     3      27.810  24.267   3.111  1.00  0.00      M1  
ATOM     29  CA  ASP     3      27.299  22.245   2.632  1.00  0.00      M1  
ATOM     30  CB  ASP     3      28.503  21.852   3.505  1.00  0.00      M1  
ATOM     31  CG  ASP     3      29.779  21.829   2.716  1.00  0.00      M1  
ATOM     32  OD1 ASP     3      29.879  22.171   1.551  1.00  0.00      M1  
ATOM     33  OD2 ASP     3      30.759  21.495   3.319  1.00  0.00      M1  
ATOM     34  C   ASP     3      26.084  21.485   3.155  1.00  0.00      M1  
ATOM     35  O   ASP     3      25.834  20.338   2.764  1.00  0.00      M1  
ATOM     36  N   PHE     4      25.348  22.133   4.032  1.00  0.00      M1  
ATOM     37  H   PHE     4      25.576  23.076   4.278  1.00  0.00      M1  
ATOM     38  CA  PHE     4      24.187  21.495   4.664  1.00  0.00      M1  
ATOM     39  CB  PHE     4      23.774  22.291   5.885  1.00  0.00      M1  
ATOM     40  CG  PHE     4      23.261  23.671   5.594  1.00  0.00      M1  
ATOM     41  CD1 PHE     4      22.050  23.813   5.002  1.00  0.00      M1  
ATOM     42  CD2 PHE     4      23.943  24.792   5.980  1.00  0.00      M1  
ATOM     43  CE1 PHE     4      21.500  25.095   4.714  1.00  0.00      M1  
ATOM     44  CE2 PHE     4      23.360  26.080   5.725  1.00  0.00      M1  
ATOM     45  CZ  PHE     4      22.186  26.205   5.078  1.00  0.00      M1  
ATOM     46  C   PHE     4      23.006  21.365   3.706  1.00  0.00      M1  
ATOM     47  O   PHE     4      22.030  20.661   3.995  1.00  0.00      M1  
ATOM     48  N   CYS     5      23.111  22.047   2.589  1.00  0.00      M1  
ATOM     49  H   CYS     5      23.837  22.726   2.483  1.00  0.00      M1  
ATOM     50  CA  CYS     5      22.174  21.827   1.482  1.00  0.00      M1  
ATOM     51  CB  CYS     5      22.342  22.861   0.400  1.00  0.00      M1  
ATOM     52  SG  CYS     5      22.120  24.593   0.905  1.00  0.00      M1  
ATOM     53  C   CYS     5      22.204  20.461   0.802  1.00  0.00      M1  
ATOM     54  O   CYS     5      21.259  20.083   0.099  1.00  0.00      M1  
ATOM     55  N   LEU     6      23.289  19.742   1.021  1.00  0.00      M1  
ATOM     56  H   LEU     6      24.042  20.123   1.557  1.00  0.00      M1  
ATOM     57  CA  LEU     6      23.415  18.381   0.487  1.00  0.00      M1  
ATOM     58  CB  LEU     6      24.880  18.078   0.129  1.00  0.00      M1  
ATOM     59  CG  LEU     6      25.591  19.031  -0.805  1.00  0.00      M1  
ATOM     60  CD1 LEU     6      26.954  18.478  -1.142  1.00  0.00      M1  
ATOM     61  CD2 LEU     6      24.793  19.235  -2.068  1.00  0.00      M1  
ATOM     62  C   LEU     6      22.960  17.225   1.364  1.00  0.00      M1  
ATOM     63  O   LEU     6      22.954  16.068   0.927  1.00  0.00      M1  
ATOM     64  OXT LEU     6      22.589  17.547   2.588  1.00  0.00      M1  
ATOM     65  HT1 CYS     1      19.779  28.460  -1.528  1.00  0.00      M2  
ATOM     66  HT2 CYS     1      19.114  27.453  -0.428  1.00  0.00      M2  
ATOM     67  N   CYS     1      19.971  27.749  -0.850  1.00  0.00      M2  
ATOM     68  HT3 CYS     1      20.580  28.116  -0.147  1.00  0.00      M2  
ATOM     69  CA  CYS     1      20.614  26.605  -1.508  1.00  0.00      M2  
ATOM     70  CB  CYS     1      19.915  25.349  -1.035  1.00  0.00      M2  
ATOM     71  SG  CYS     1      20.136  24.920   0.720  1.00  0.00      M2  
ATOM     72  C   CYS     1      20.748  26.451  -3.024  1.00  0.00      M2  
ATOM     73  O   CYS     1      21.370  25.498  -3.515  1.00  0.00      M2  
ATOM     74  OXT CYS     1      20.162  27.391  -3.743  1.00  0.00      M2  
ATOM     75  OH2 TIP     1      21.415  27.234  -7.180  1.00  0.00      SOLV
ATOM     76  H1  TIP     1      21.087  27.275  -6.282  1.00  0.00      SOLV
ATOM     77  H2  TIP     1      22.339  27.004  -7.088  1.00  0.00      SOLV
END

surface 
   mode=access rh2o=1.4 accuracy=0.05 selection=( all )
end

show element ( rmsd ) ( attr rmsd > 0 )

surface
   mode=contact selection= ( all  ) 
end

show element ( rmsd ) ( attr rmsd > 0  )

surface 
   mode=access radius=vdw rh2o=1.4 accuracy=0.05 selection=( all )
end

show element ( rmsd ) ( attr rmsd > 0  )


stop
