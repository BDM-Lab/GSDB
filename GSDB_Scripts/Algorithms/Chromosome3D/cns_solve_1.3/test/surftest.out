          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:09 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks SURFTEST.INP 
 CNSsolve>remarks TEST OF THE RICHARD'S ACCESSIBLE SURFACE AND CONTACT PROPERTIES USING 
 CNSsolve>remarks A SHORTENED PTI TO SPEED THE RUN. 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>topology @CNS_TOPPAR:protein.top  @CNS_TOPPAR:water.top end 
 ASSFIL: file protein.top opened.
 RTFRDR>remarks file toppar/protein.top 
 RTFRDR>remarks   protein topology with polar hydrogens 
 RTFRDR> 
 RTFRDR>! 
 RTFRDR>! Please cite the following reference when using these parameters: 
 RTFRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 RTFRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 RTFRDR>!  Acta Cryst. A47, 392-400. 
 RTFRDR>! 
 RTFRDR>! 
 RTFRDR> 
 RTFRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 RTFRDR>set message=off echo=off end 
 RTFRDR> 
 RTFRDR> 
 RTFRDR>  @CNS_TOPPAR:water.top end 
 ASSFIL: file water.top opened.
 RTFRDR>remarks file toppar/water.top 
 RTFRDR>remarks   water topology for crystallographic structure determination 
 RTFRDR>remarks   based on Jorgensen Tip3p water model 
 RTFRDR> 
 RTFRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 RTFRDR>set message=off echo=off end 
 RTFRDR> 
 RTFRDR> end 
 CNSsolve>parameter @CNS_TOPPAR:protein.param  @CNS_TOPPAR:water.param end 
 ASSFIL: file protein.param opened.
 PARRDR>remarks file toppar/protein.param 
 PARRDR>remarks   protein parameters with Lennard-Jones van der Waals parameters 
 PARRDR>remarks   for crystallographic structure determination 
 PARRDR> 
 PARRDR>! 
 PARRDR>! Please cite the following reference when using these parameters: 
 PARRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 PARRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 PARRDR>!  Acta Cryst. A47, 392-400. 
 PARRDR>! 
 PARRDR>! 
 PARRDR> 
 PARRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 PARRDR>set message=off echo=off end 
 PARRDR> 
 PARRDR>  @CNS_TOPPAR:water.param end 
 ASSFIL: file water.param opened.
 PARRDR>remarks file toppar/water.param 
 PARRDR>remarks   water parameters for structure determination 
 PARRDR>remarks 
 PARRDR>remarks   available: Jorgensen TIPS3P model 
 PARRDR> 
 PARRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 PARRDR>set message=off echo=off end 
 PARRDR> 
 PARRDR> end 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>   name=m1 
 SEGMENT>   chain 
 CHAIN>      @CNS_TOPPAR:protein.link 
 ASSFIL: file protein.link opened.
 CHAIN>remarks file toppar/protein.link 
 CHAIN>remarks 
 CHAIN>remarks this is a macro to define standard protein peptide bonds 
 CHAIN>remarks and termini to generate a protein sequence. 
 CHAIN> 
 CHAIN>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 CHAIN>set message=off echo=off end 
 CHAIN>      sequence 
 SEQUENCE>         arg pro asp phe cys leu 
 SEQUence-element (terminate with END) =      end 
 SEGMNT:     6 residues were inserted into segment "M1  "
 CHAIN>   end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         64(MAXA=     1000000)  NBOND=         65(MAXB=     1000000)
 -> NTHETA=        93(MAXT=     2000000)  NGRP=           8(MAXGRP=   1000000)
 -> NPHI=          39(MAXP=     2000000)  NIMPHI=        32(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>  name=m2 
 SEGMENT>  chain 
 CHAIN>     @CNS_TOPPAR:protein.link 
 ASSFIL: file protein.link opened.
 CHAIN>remarks file toppar/protein.link 
 CHAIN>remarks 
 CHAIN>remarks this is a macro to define standard protein peptide bonds 
 CHAIN>remarks and termini to generate a protein sequence. 
 CHAIN> 
 CHAIN>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 CHAIN>set message=off echo=off end 
 CHAIN>     sequence cys end 
 SEGMNT:     1 residues were inserted into segment "M2  "
 CHAIN>  end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         74(MAXA=     1000000)  NBOND=         74(MAXB=     1000000)
 -> NTHETA=       106(MAXT=     2000000)  NGRP=          11(MAXGRP=   1000000)
 -> NPHI=          44(MAXP=     2000000)  NIMPHI=        34(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>patch disu reference 1 ( atom m1 5 * ) reference 2 ( atom m2 1 * )  end 
 SELRPN:      7 atoms have been selected out of     74
 SELRPN:     10 atoms have been selected out of     74
 Status of internal molecular topology database:
 -> NATOM=         74(MAXA=     1000000)  NBOND=         75(MAXB=     1000000)
 -> NTHETA=       108(MAXT=     2000000)  NGRP=          12(MAXGRP=   1000000)
 -> NPHI=          47(MAXP=     2000000)  NIMPHI=        34(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>    name=solv 
 SEGMENT>    molecule 
 MOLECULE>       name=TIP  number=1 
 MOLECULE>    end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         77(MAXA=     1000000)  NBOND=         77(MAXB=     1000000)
 -> NTHETA=       109(MAXT=     2000000)  NGRP=          13(MAXGRP=   1000000)
 -> NPHI=          47(MAXP=     2000000)  NIMPHI=        34(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>coor 
 COOR>ATOM      1  HT1 ARG     1      26.033  27.513  -3.554  1.00  0.00      M1 
 COOR>ATOM      2  HT2 ARG     1      26.877  28.308  -2.405  1.00  0.00      M1 
 CNSsolve> 
 CNSsolve>surface 
 SURFAC>   mode=access rh2o=1.4 accuracy=0.05 selection=( all ) 
 SELRPN:     77 atoms have been selected out of     77
 SURFAC>end 
 SURFAC: half the sigma value being used for atomic radii.
 SURFAC: ACCEssible surface area
 SURFAC: ACCUracy= 0.05 RH2O=  1.40
         Z-grid=  0.16 number-of-Z-sections=  121
         measures-of-arc=   615 and  1257
 CNSsolve> 
 CNSsolve>show element ( rmsd ) ( attr rmsd > 0 ) 
 SELRPN:     63 atoms have been selected out of     77
 ( M1   ARG  1    CB   )    3.0523    
 ( M1   ARG  1    CG   )    9.3788    
 ( M1   ARG  1    CD   )    5.7440    
 ( M1   ARG  1    NE   )   0.39040    
 ( M1   ARG  1    CZ   )    20.415    
 ( M1   ARG  1    NH1  )    27.691    
 ( M1   ARG  1    HH11 )   0.32573    
 ( M1   ARG  1    HH12 )    3.8042    
 ( M1   ARG  1    NH2  )    23.691    
 ( M1   ARG  1    HH21 )    3.2974    
 ( M1   ARG  1    C    )    3.7643    
 ( M1   ARG  1    O    )    16.579    
 ( M1   ARG  1    HT1  )    1.8519    
 ( M1   ARG  1    HT2  )    2.6927    
 ( M1   ARG  1    N    )    26.708    
 ( M1   ARG  1    HT3  )    1.7341    
 ( M1   ARG  1    CA   )    16.550    
 ( M1   PRO  2    CD   )    15.026    
 ( M1   PRO  2    CA   )    14.735    
 ( M1   PRO  2    CB   )    11.886    
 ( M1   PRO  2    CG   )    29.150    
 ( M1   PRO  2    C    )   0.19508E-03
 ( M1   ASP  3    N    )   0.14288    
 ( M1   ASP  3    H    )    2.4591    
 ( M1   ASP  3    CA   )   0.31114    
 ( M1   ASP  3    CB   )    32.490    
 ( M1   ASP  3    CG   )    17.925    
 ( M1   ASP  3    OD1  )    26.914    
 ( M1   ASP  3    OD2  )    36.700    
 ( M1   ASP  3    C    )   0.60049    
 ( M1   ASP  3    O    )    6.4603    
 ( M1   PHE  4    CA   )    14.825    
 ( M1   PHE  4    CB   )    33.294    
 ( M1   PHE  4    CG   )    1.5851    
 ( M1   PHE  4    CD1  )    12.323    
 ( M1   PHE  4    CD2  )    21.774    
 ( M1   PHE  4    CE1  )    20.171    
 ( M1   PHE  4    CE2  )    23.480    
 ( M1   PHE  4    CZ   )    24.484    
 ( M1   PHE  4    O    )    17.132    
 ( M1   CYS  5    CA   )    7.1278    
 ( M1   CYS  5    C    )    1.0672    
 ( M1   CYS  5    O    )    25.959    
 ( M1   CYS  5    CB   )    1.5947    
 ( M1   LEU  6    CA   )    6.0464    
 ( M1   LEU  6    CB   )    15.125    
 ( M1   LEU  6    CG   )    1.1060    
 ( M1   LEU  6    CD1  )    58.073    
 ( M1   LEU  6    CD2  )    34.732    
 ( M1   LEU  6    C    )    14.759    
 ( M1   LEU  6    O    )    35.306    
 ( M1   LEU  6    OXT  )    23.883    
 ( M2   CYS  1    CB   )    30.748    
 ( M2   CYS  1    SG   )    19.761    
 ( M2   CYS  1    C    )    7.7467    
 ( M2   CYS  1    O    )    6.3248    
 ( M2   CYS  1    OXT  )    26.225    
 ( M2   CYS  1    HT1  )    1.6625    
 ( M2   CYS  1    HT2  )    1.3375    
 ( M2   CYS  1    N    )    27.614    
 ( M2   CYS  1    HT3  )    1.7758    
 ( M2   CYS  1    CA   )    5.6333    
 ( SOLV TIP  1    OH2  )    69.735    
 CNSsolve> 
 CNSsolve>surface 
 SURFAC>   mode=contact selection= ( all  ) 
 SELRPN:     77 atoms have been selected out of     77
 SURFAC>end 
 SURFAC: half the sigma value being used for atomic radii.
 SURFAC: CONTact area
 SURFAC: ACCUracy= 0.05 RH2O=  1.60
         Z-grid=  0.18 number-of-Z-sections=  110
         measures-of-arc=   639 and  1432
 CNSsolve> 
 CNSsolve>show element ( rmsd ) ( attr rmsd > 0  ) 
 SELRPN:     61 atoms have been selected out of     77
 ( M1   ARG  1    CB   )   0.65194    
 ( M1   ARG  1    CG   )    2.5375    
 ( M1   ARG  1    CD   )    1.5874    
 ( M1   ARG  1    NE   )   0.59569E-01
 ( M1   ARG  1    CZ   )    5.8415    
 ( M1   ARG  1    NH1  )    6.5387    
 ( M1   ARG  1    HH11 )   0.12794E-01
 ( M1   ARG  1    HH12 )   0.26987    
 ( M1   ARG  1    NH2  )    5.3708    
 ( M1   ARG  1    HH21 )   0.27616    
 ( M1   ARG  1    C    )    1.0920    
 ( M1   ARG  1    O    )    3.6529    
 ( M1   ARG  1    HT1  )   0.15011    
 ( M1   ARG  1    HT2  )   0.19146    
 ( M1   ARG  1    N    )    6.7837    
 ( M1   ARG  1    HT3  )   0.11190    
 ( M1   ARG  1    CA   )    5.5013    
 ( M1   PRO  2    CD   )    4.6371    
 ( M1   PRO  2    CA   )    4.8625    
 ( M1   PRO  2    CB   )    3.7551    
 ( M1   PRO  2    CG   )    9.3738    
 ( M1   ASP  3    H    )   0.21359    
 ( M1   ASP  3    CA   )   0.20272    
 ( M1   ASP  3    CB   )    9.7696    
 ( M1   ASP  3    CG   )    5.2719    
 ( M1   ASP  3    OD1  )    5.9866    
 ( M1   ASP  3    OD2  )    9.1884    
 ( M1   ASP  3    C    )   0.12264    
 ( M1   ASP  3    O    )    1.3410    
 ( M1   PHE  4    CA   )    4.7412    
 ( M1   PHE  4    CB   )    10.947    
 ( M1   PHE  4    CG   )   0.49455    
 ( M1   PHE  4    CD1  )    3.5947    
 ( M1   PHE  4    CD2  )    6.5918    
 ( M1   PHE  4    CE1  )    6.0920    
 ( M1   PHE  4    CE2  )    7.3428    
 ( M1   PHE  4    CZ   )    7.6357    
 ( M1   PHE  4    O    )    3.9078    
 ( M1   CYS  5    CA   )    2.1461    
 ( M1   CYS  5    C    )   0.27928    
 ( M1   CYS  5    O    )    6.0468    
 ( M1   CYS  5    CB   )   0.19092    
 ( M1   LEU  6    CA   )    1.9163    
 ( M1   LEU  6    CB   )    4.5947    
 ( M1   LEU  6    CG   )   0.11074    
 ( M1   LEU  6    CD1  )    18.376    
 ( M1   LEU  6    CD2  )    10.456    
 ( M1   LEU  6    C    )    4.3737    
 ( M1   LEU  6    O    )    8.6563    
 ( M1   LEU  6    OXT  )    5.4054    
 ( M2   CYS  1    CB   )    9.9981    
 ( M2   CYS  1    SG   )    5.1380    
 ( M2   CYS  1    C    )    2.0732    
 ( M2   CYS  1    O    )    1.3435    
 ( M2   CYS  1    OXT  )    6.1544    
 ( M2   CYS  1    HT1  )   0.10041    
 ( M2   CYS  1    HT2  )   0.87065E-01
 ( M2   CYS  1    N    )    6.8605    
 ( M2   CYS  1    HT3  )   0.79797E-01
 ( M2   CYS  1    CA   )    1.6246    
 ( SOLV TIP  1    OH2  )    18.553    
 CNSsolve> 
 CNSsolve>surface 
 SURFAC>   mode=access radius=vdw rh2o=1.4 accuracy=0.05 selection=( all ) 
 SELRPN:     77 atoms have been selected out of     77
 SURFAC>end 
 SURFAC: half the vdw value being used for atomic radii.
 SURFAC: ACCEssible surface area
 SURFAC: ACCUracy= 0.05 RH2O=  1.40
         Z-grid=  0.16 number-of-Z-sections=  122
         measures-of-arc=   686 and  1557
 CNSsolve> 
 CNSsolve>show element ( rmsd ) ( attr rmsd > 0  ) 
 SELRPN:     52 atoms have been selected out of     77
 ( M1   ARG  1    CB   )    2.0804    
 ( M1   ARG  1    CG   )    8.9902    
 ( M1   ARG  1    CD   )    5.2637    
 ( M1   ARG  1    NE   )   0.75479E-01
 ( M1   ARG  1    CZ   )    23.980    
 ( M1   ARG  1    NH1  )    31.665    
 ( M1   ARG  1    NH2  )    26.337    
 ( M1   ARG  1    C    )    3.7804    
 ( M1   ARG  1    O    )    15.629    
 ( M1   ARG  1    N    )    33.998    
 ( M1   ARG  1    CA   )    19.934    
 ( M1   PRO  2    CD   )    15.133    
 ( M1   PRO  2    CA   )    16.666    
 ( M1   PRO  2    CB   )    12.226    
 ( M1   PRO  2    CG   )    30.987    
 ( M1   ASP  3    H    )   0.74865    
 ( M1   ASP  3    CA   )   0.28442    
 ( M1   ASP  3    CB   )    34.912    
 ( M1   ASP  3    CG   )    21.331    
 ( M1   ASP  3    OD1  )    25.764    
 ( M1   ASP  3    OD2  )    38.486    
 ( M1   ASP  3    C    )   0.46313    
 ( M1   ASP  3    O    )    5.2237    
 ( M1   PHE  4    CA   )    15.700    
 ( M1   PHE  4    CB   )    36.309    
 ( M1   PHE  4    CG   )    1.4517    
 ( M1   PHE  4    CD1  )    12.926    
 ( M1   PHE  4    CD2  )    22.547    
 ( M1   PHE  4    CE1  )    21.411    
 ( M1   PHE  4    CE2  )    25.758    
 ( M1   PHE  4    CZ   )    26.428    
 ( M1   PHE  4    O    )    16.557    
 ( M1   CYS  5    CA   )    7.5781    
 ( M1   CYS  5    C    )    1.1918    
 ( M1   CYS  5    O    )    25.999    
 ( M1   CYS  5    CB   )   0.83379    
 ( M1   LEU  6    CA   )    6.9420    
 ( M1   LEU  6    CB   )    15.852    
 ( M1   LEU  6    CG   )   0.33226    
 ( M1   LEU  6    CD1  )    62.198    
 ( M1   LEU  6    CD2  )    33.566    
 ( M1   LEU  6    C    )    18.078    
 ( M1   LEU  6    O    )    35.949    
 ( M1   LEU  6    OXT  )    23.536    
 ( M2   CYS  1    CB   )    34.199    
 ( M2   CYS  1    SG   )    18.780    
 ( M2   CYS  1    C    )    8.0688    
 ( M2   CYS  1    O    )    5.3920    
 ( M2   CYS  1    OXT  )    26.480    
 ( M2   CYS  1    N    )    32.257    
 ( M2   CYS  1    CA   )    6.7764    
 ( SOLV TIP  1    OH2  )    75.204    
 CNSsolve> 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8251632 bytes
           Maximum dynamic memory overhead:           464 bytes
           Program started at: 16:08:09 on 05-Jul-2018
           Program stopped at: 16:08:09 on 05-Jul-2018
           CPU time used:       0.0428 seconds
          ============================================================
